src/pages/communs/index.tsx

```diff
@@ -100,16 +102,22 @@ export default function Landing() {
             collaboration à d&apos;autres projets de communs numériques.
           </p>
           <p className="mb-[42px]">
-            <Strong>Comment :</Strong> en utilisant les apprentissages du
-            premier lot, contribuer à d&apos;autres projets de communs
-            numériques autour des thématiques de La Suite numérique,
-            c&apos;est-à-dire tout outil collaboratif qui peut aider les agents
-            publics à faire leur travail de manière plus efficace.
+            <Strong>Comment :</Strong> en utilisant les apprentissages de la
+            première phase, contribuer à d’autres projets de Communs Numériques
+            éligibles au Fonds.
           </p>
         </div>
       </div>
+      <div className="w-full items-center flex flex-col">
+        <h2 className="text-2xl text-[1.75rem] sm:text-4xl font-bold px-4 pb-[2rem] max-w-[31em] text-center text-balance">
+          Processus de sélection
+        </h2>
+        <Image src="/financement-communs.svg" width={800} height={640} alt="" />
```

Comment:
we'll need a way to mention the info contained in this image with text. I can work on that later or you can already discuss it with Johann. But having actual text visible to all users would be great because I'm not sure that image would be clearly readable on mobile devices anyway.

Reply:
Yes ofc, I have asked Camille and Johann to work on it, to provide the relevant text